core/files/css/base.php

```diff
@@ -673,6 +678,38 @@ protected function add_control_style_rules( array $control, array $values, array
 		);
 	}
 
+	/**
+	 * Get Control Global Default Value
+	 *
+	 * If the control has a global default value, and the corresponding global default setting is enabled, this method
+	 * fetches and returns the global default value. Otherwise, it returns null.
+	 *
+	 * @since 3.7.0
+	 * @access private
+	 *
+	 * @param $control
+	 * @return string|null
+	 */
+	private function get_control_global_default_value( $control ) {
+		if ( ! empty( $control['global']['default'] ) ) {
+			// If the control value is empty, and the control has a global default set, fetch the global value and use it.
+			$global_enabled = false;
```

Comment:
What's the different between `false` to `null` here?

Reply:
The difference is that the following methods return booleans that override this variable's value. So I prefer to instantiate it as a boolean as well to keep the typing identical.